### ISSUE TITLE
NVDA does not announce dynamically-injected checkbox description text in Firefox + browse mode

### ISSUE THREAD

**Issue by JamesCatt** [nvaccess/nvda#11739]
Here's a demo page for reproducing this issue: https://jamescatt.ca/test/ariadescribedby/

When a checkbox has `aria-describedby` pointing to another element on the page, and text is injected inside that element with Javascript, the description is not announced by NVDA with Firefox while in browse mode. This occurs whether tabbing to the checkbox or browsing with arrow keys. If you press Insert + Spacebar and then tab to the checkbox, it is correctly announced. Some additional notes:

- Firefox developer tools correctly report the text as the accessible description for the checkbox
- Issue does not occur with NVDA + Chrome
- Issue does not occur with JAWS + Firefox

### Steps to reproduce:

1. Insert text into the element referenced by `aria-describedby` with Javascript.
2. Navigate to the checkbox either by setting focus with Javascript, tabbing, or using arrow keys.

### Actual behavior:
The description text is not announced when the checkbox is focused.

### Expected behavior:
The description text should be announced.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
2020.2

#### Windows version:
10

#### Name and version of other software in use when reproducing the issue:
Firefox 81, 82 beta

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
I believe earlier versions have this issue as well, but I haven't checked.

#### If addons are disabled, is your problem still occuring?
I have no addons running.

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No, I'm not sure what this is.

**Comment by Adriani90** [comment]
cc: @jcsteh, @MarcoZehe

**Comment by jcsteh** [comment]
Duplicate of #9400.
